js/web/lib/wasm/jsep/webgpu/ops/pool.ts

```diff
@@ -332,13 +334,13 @@ const globalPoolAttributes = {
   strides: [],
   pads: [],
   storageOrder: 0,
-  dilations: [],
-  cacheKey: ''
+  dilations: []
 };
 
 export const parseGlobalAveragePoolAttributes = (attributes: Record<string, unknown>): AveragePoolAttributes => {
   const format = attributes.format as FormatAttributes['format'];
-  return {format, ...globalPoolAttributes, cacheKey: format};
+  const attributesWithoutCacheKey = {format, ...globalPoolAttributes};
+  return {...attributesWithoutCacheKey, cacheKey: createShaderKeyFromAttributes(attributesWithoutCacheKey)};
```

Comment:
same to GlobalMaxPool. the only info we care about is format.

Reply:
done